src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("the truck started it`s work");
```

Comment:
Replace the backtick (`) with an apostrophe (') in `it's` to use the correct punctuation for the contraction of `it is`.